### ISSUE TITLE
Option for Video Player Size to fill Window

### ISSUE THREAD

**Issue by SmoresDecore** [TeamPiped/Piped#3477]
### Describe the feature

An option or toggle to have the player size scale with the window size so it is as big as possible instead of the default size.

### Why would this be useful to add?

This would make it so videos can be as big as possible without having to go in fullscreen or as small as wanted by resizing the window itself

### Concept(s)

Basically copy this script: https://greasyfork.org/en/scripts/811-resize-yt-to-window-size

### Acknowledgements

- [X] I have searched the existing issues and this is **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this form.

**Comment by Keefaetana** [comment]
I agree with your request. Until then, I've been using this script: https://openuserjs.org/scripts/jnaskali/Piped_full-width .

**Comment by Bnyro** [comment]
Whoever wrote this script could just have been creating a pull request to add the option here :shrug:
